src/main/scala/org/broadinstitute/dsde/rawls/model/MethodRepoModel.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.rawls.model
+
+import com.wordnik.swagger.annotations.{ApiModelProperty, ApiModel}
+import org.joda.time.DateTime
+
+import scala.annotation.meta.field
+
+object AgoraEntityType extends Enumeration {
+  type EntityType = Value
+  val Task = Value("Task")
+  val Workflow = Value("Workflow")
+  val Configuration = Value("Configuration")
+}
+
+@ApiModel(value = "AgoraEntity")
+case class AgoraEntity(
+                        @(ApiModelProperty@field)(required = false, value = "The namespace to which the method belongs")
```

Comment:
Maybe the docs should say "method/config" instead of "method", since the entity could be either?